packages/manager/apps/telecom/src/app/telecom/telephony/repayments/repayments.html

```diff
@@ -0,0 +1,69 @@
+<section class="telephony-group-line-repayments">
+    <section class="widget-presentation">
+        <div class="row">
+            <div class="col-md-6">
+                <h4 data-translate="telephony_repayments_repayments"></h4>
+                <dl>
+                    <dt
+                        data-translate="telephony_repayments_pending_title"
+                    ></dt>
+                    <dd data-ng-bind="$ctrl.pendingAmountText"></dd>
+                    <dt
+                        data-translate="telephony_repayments_available_title"
+                    ></dt>
+                    <dd data-ng-bind="$ctrl.availableAmountText"></dd>
+                </dl>
+            </div>
+            <div class="col-md-6">
+                <h4 data-translate="telephony_repayments_costs"></h4>
+                <dl>
+                    <dt data-translate="telephony_repayments_costs_title"></dt>
+                    <dd data-ng-bind="$ctrl.costsAmountText"></dd>
+                </dl>
+            </div>
+        </div>
+
+        <oui-message
+            type="info"
+            class="mt-3"
+            style="display: inline-block !important"
+        >
+            <span
+                data-ng-if="$ctrl.hasReachedRepaymentAmountThreshold"
+                data-translate="telephony_repayments_message_above_threshold"
+                data-translate-values="{ amount: $ctrl.availableAmountText }"
+            ></span>
+            <span
+                data-ng-if="!$ctrl.hasReachedRepaymentAmountThreshold"
+                data-translate="telephony_repayments_message_under_threshold"
+                data-translate-values="{ amount: $ctrl.repaymentAmountThresholdText }"
+            ></span>
+        </oui-message>
+
+        <div data-ng-if="$ctrl.isLoadingIban">
+            <oui-spinner size="s"></oui-spinner>
+        </div>
+        <div
+            data-ng-if="$ctrl.iban"
+            class="d-flex flex-column align-items-start mt-2 mb-3"
+        >
+            <h3 data-translate="telephony_repayments_message_bank_account"></h3>
+            <label class="oui-field__label">IBAN</label>
+            <div
+                class="oui-box oui-box_light mb-2"
+                style="padding: .5rem;"
```

Comment:
consider to use the bootstrap class instead of custom style to have a consistency values across project.
You can use: `p-0, p-1, p-2, ...`

```suggestion
                class="oui-box oui-box_light mb-2 p-2"
```

Reply:
It's applied